comictaggerlib/crediteditorwindow.py

```diff
@@ -75,7 +75,9 @@ def __init__(self, parent: QtWidgets.QWidget, mode: int, credit: Credit) -> None
                 self.cbRole.setCurrentIndex(i)
 
         if credit.language != "":
-            i = self.cbLanguage.findText(credit.language)
+            i = self.cbLanguage.findData(credit.language, QtCore.Qt.ItemDataRole.UserRole) or self.cbLanguage.findText(
+                credit.language
+            )
```

Comment:
If findData doesn't find something it returns -1 which is truthy `bool(-1) is True` so this won't work you have to separate it out into it's own if statement

Reply:
I always forget -1 isn't false...